2.5/alpine3.11/Dockerfile

```diff
@@ -127,6 +127,7 @@ ENV GEM_HOME /usr/local/bundle
 ENV BUNDLE_SILENCE_ROOT_WARNING=1 \
 	BUNDLE_APP_CONFIG="$GEM_HOME"
 ENV PATH $GEM_HOME/bin:$PATH
+ENV LANG C.UTF-8
```

Comment:
This isn't really related to the block of settings it's been stuck in the middle of -- could we move it up higher in the file?  Probably on a line by itself just before `ENV RUBY_MAJOR`?

Reply:
I moved this line to the just before `ENV RUBY_MAJOR` 👍